pkg/processor/runtime/dotnetcore/test/_outputter/outputter.cs

```diff
@@ -28,6 +28,12 @@ public class Account
 
 public class nuclio
 {
+    public void InitContext(Context context){
+        context.Name = "Test";
+        context.IsInitialized = true;
+        context.UserData.Add("a", "b");
```

Comment:
Missing initializing `UserData`.
`context.UserData = new Dictionary<string, object>();`

Reply:
I was on the fence about whether to initialize UserData in the Context class constructor or leave it up to the user. My initial thought was that there is no need to use memory if the function doesn't need to store user data. What do you think?

Reply:
@ballcoach12 you may leave it up to the user. just add the initilization here